code/modules/admin/verbs/adminfun.dm

```diff
@@ -153,8 +153,17 @@ ADMIN_VERB(polymorph_all, R_ADMIN, "Polymorph All", "Applies the effects of the
 
 	message_admins("Mass polymorph started by [who_did_it] is complete.")
 
-ADMIN_VERB_AND_CONTEXT_MENU(admin_smite, R_ADMIN|R_FUN, "Smite", "Smite a player with divine power.", ADMIN_CATEGORY_FUN, mob/living/target in world)
```

Comment:
This is necessary so that it appears in the context menu of mob/living/target for admins. Otherwise, this won't appear in the context menu and it'll be extremely tedious to use the Smite verb. You should undo this for the rest of them too.